dbt_subprojects/solana/models/_sector/dex/dex_solana_base_trades.sql

```diff
@@ -53,10 +60,10 @@ SELECT
       , tx_index
 FROM
       {{ dex }}
-{% if is_incremental() %}
 WHERE
+      {% if is_incremental() %}
       {{incremental_predicate('block_time')}}
-{% endif %}
+      {% endif %}
```

Comment:
you need the WHERE wrapped in here still like before or it'll fail on historical run

Reply:
can you just revert back to original instead?

Reply:
tbh I would't know how to do that, can I just git reverse specific rows of code somehow?

Reply:
if you tell me that's possible without a huge hassle I can prob vibecode that, just never looked into it

Reply:
i'm just referring to `git revert <commit_number>` but i see you have various time filter commits 😅 
probably easier to just manually clean up the filters again with another commit